src/modules/universal-discovery/components/finder/finder.component.js

```diff
@@ -42,9 +43,16 @@ export default class FinderComponent extends Component {
             this.loadPreselectedData(this.props.preselectedLocation);
         } else {
             this.setDefaultActiveLocations();
-            this.props.findLocationsByParentLocationId(
-                { ...this.props.restInfo, parentLocationId: this.props.startingLocationId },
-                this.updateLocationsData
+            this.props.loadLocation(
+                { ...this.props.restInfo, locationId: this.props.startingLocationId },
+                (response) => {
+                    this.startingLocation = response.View.Result.searchHits.searchHit[0].value.Location;
```

Comment:
I think this will fail if `startingLocationId` will be 1. In this case, there is no location.

Reply:
I checked changes locally and I have `TypeError: Cannot read property 'value' of undefined` for `startingLocationId: 1`; for others it works.